source/sphinx_extensions/command_block/extension.py

```diff
@@ -40,17 +63,34 @@ class CommandBlockDirective(docutils.parsers.rst.Directive):
     has_content = True
```

Comment:
From the docutils docs:

```
    - `has_content`: A boolean; True if content is allowed.  Client
      code must handle the case where content is required but not
      supplied (an empty content list will be supplied).
```

So looks like for our purposes, this does nothing. Also, the `assert_has_content` uses that attr, which does us no good.